documentation/book/con-define-cluster-roles.adoc

```diff
@@ -0,0 +1,41 @@
+[id='define-cluster-roles-{context}']
+= Define `ClusterRoles`
+
+The Cluster Operator needs to operate using `ClusterRoles` that gives access to the necessary resources.
+Depending on the {ProductPlatformName} cluster setup, a cluster administrator may need to create the `ClusterRoles`.
+
+[NOTE]
+Cluster administrator rights are only needed for the creation of the `ClusterRoles`. The Cluster Operator will not run under the cluster admin account.
+
+The `ClusterRoles` follow the _principle of least privilege_ and contain only those privileges needed by the Cluster Operator to operate Kafka, Kafka Connect, and Zookeeper clusters. The first set of assigned privileges allow the Cluster Operator to manage {ProductPlatformName} resources such as `StatefulSets`, `Deployments`, `Pods`, and `ConfigMaps`.
+
+[source,yaml,options="nowrap"]
+.Example `Role` for the Cluster Operator
+----
+include::examples/install/cluster-operator/02-ClusterRole-strimzi-cluster-operator-role.yaml[]
+----
+
+The `strimzi-kafka-broker` `ClusterRole` represents the access needed by the init container in Kafka pods that is used for the rack feature. As described in the xref:delegated-privileges-{context}[Delegated Privileges] section, this role is also needed by the Cluster Operator in order to be able to delegate this access.
+
+[source,yaml,options="nowrap"]
+.`ClusterRole` for the Cluster Operator allowing it to delegate access to {ProductPlatformName} nodes to the Kafka broker pods
+----
+# These are the privileges needed by the init container
```

Comment:
I'm not sure I understand why do we have these comments here in the docu. You already explained them above, so I think they can b e remove here as well as in the following source section.